controllers/templates/backup-templates.go

```diff
@@ -102,128 +102,158 @@ spec:
-                    name: host-var
-                    subPath: lib/containers
                   - mountPath: /host/var/lib/cni
-                    name: host-var
-                    subPath: lib/cni
+                    name: host-var-lib-cni
                     readOnly: true
+                  - mountPath: /host/var/lib/containers
+                    name: host-var-lib-containers
                   - mountPath: /host/var/lib/etcd
-                    name: host-var
-                    subPath: lib/etcd
+                    name: host-var-lib-etcd
                     readOnly: true
                   - mountPath: /host/var/lib/kubelet
                     name: host-var-lib-kubelet
                     readOnly: true
                   - mountPath: /host/var/lib/ovn-ic
-                    name: host-var
-                    subPath: lib/ovn-ic
+                    name: host-var-lib-ovn-ic
                     readOnly: true
+                  - mountPath: /host/var/recovery
+                    name: host-var-recovery
                   - mountPath: /host/var/tmp
-                    name: host-var
-                    subPath: tmp
-                  - mountPath: /host/boot
-                    name: host-boot
-                    readOnly: true
-                  - mountPath: /host/run
-                    name: host-run
-                  - mountPath: /host/sysroot
-                    name: host-sysroot
+                    name: host-var-tmp
                   - mountPath: /host/sys/fs/cgroup
                     name: sys-fs-cgroup
                     readOnly: true
-                  - mountPath: /host/dev/log
-                    name: host-dev-log
-                  - mountPath: /host/tmp
-                    name: host-tmp
             restartPolicy: Never
             serviceAccountName: backup-agent
             volumes:
               - emptyDir: {}
                 name: host
+              - hostPath:
+                  path: /boot
+                  type: Directory
+                name: host-boot
+              - hostPath:
+                  path: /dev/log
+                  type: Socket
+                name: host-dev-log
+              - hostPath:
+                  path: /etc
+                  type: Directory
+                name: host-etc
+              - hostPath:
+                  path: /proc
+                  type: Directory
+                name: host-proc
+              - hostPath:
+                  path: /run
+                  type: Directory
+                name: host-run
+              - hostPath:
+                  path: /sysroot
+                  type: Directory
+                name: host-sysroot
+              - hostPath:
+                  path: /tmp
+                  type: Directory
+                name: host-tmp
               - hostPath:
                   path: /usr
                   type: Directory
                 name: host-usr
```

Comment:
Do we still need this?

Reply:
It looks like we do, I have tried to remove these top level mounts and everything falls apart very quickly

Reply:
I didn't realize the two pods were using different methods to mount /etc so when I tried to remove the high level mounts from both pods that caused my problems. The backup pod does mount /etc but the precache pod just mounts a few files under /etc instead